spec/replace_browser_extension/01_main.rst

```diff
@@ -0,0 +1,540 @@
+---
+story: 60
+spec: 54
+shortcode: RBE
+title: Repalce Existing Browser Extension
+author: Dmitry Bespalov
+status: Final
+created: 2019-01-04
+---
+
+==================================
+Replace Existing Browser Extension
+==================================
+
+1. `Main`_
+2. `User Interface`_
```

Comment:
I think we should different per-platform links, as these are different product

Reply:
More different clients for the same product.
I would really like to keep them in 1 file so we can look at them in parallel. So in case the UI is different we are much more aware and deliberate about it.